help/de-de.markdown

```diff
@@ -54,31 +55,27 @@ comments: []
 <h2>Erstellung eines Screenshots</h2>
 <p>
 Zum Erstellen eines Screenshots können Sie entweder die <kbd>Drucken</kbd>-Taste
-verwenden oder sie klicken das Greenshot-Icon im Systray mit der rechten Maustaste.<br>
+verwenden oder Sie klicken das Greenshot-Icon in der Taskleiste mit der rechten Maustaste.<br>
```

Comment:
I'm not sure about changing this to "Taskleiste" as it is less specific - it seems it is called "Infobereich" nowadays? What do you think about:
```suggestion
verwenden oder Sie klicken das Greenshot-Icon im Infobereich der Taskleiste mit der rechten Maustaste.<br>
```

Reply:
btw: I also changed the english version to "notification area" instead of "systray"

Reply:
you are right, perfect